cargo/cargo_build_script.bzl

```diff
@@ -316,32 +303,43 @@ def cargo_build_script(
     build script in addition to the file generated by it.
 
     Args:
-        name (str): The target name for the underlying rule
-        crate_name (str, optional): Name of the crate associated with this build script target.
+        name (str): The target name for the underlying rule. This should be the name of the crate being compiled, optionally with a suffix of _build_script.
```

Comment:
Since we're being crisp about the distinction between package names and crate names, replace "name of the crate being compiled" with "name of the package being compiled".

Reply:
Done